Automattic/Gridicons-iOS
# Add P2 icon

Parent issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/15264

This PR updates the .proj files so it's compatible with Xcode ~12~ 11, applies the Xcode recommended project settings updates, adds the P2 icon, and updates the podspec file.

<a href="https://user-images.githubusercontent.com/1062444/100914100-37c0fd00-3498-11eb-8d4c-b7ac2545f509.png"><img src="https://user-images.githubusercontent.com/1062444/100914100-37c0fd00-3498-11eb-8d4c-b7ac2545f509.png" width="350" /></a>

### To test
1. Check out this branch
2. `rake dependencies`
3. `rake gen`
4. Build and run the Demo
5. See the new P2 icon listed in the demo
